README.md

```diff
@@ -47,7 +50,13 @@ cd deployments/docker
 docker build -t aerogear/aerogear-metrics-api .
 ```
 
-### Release
+## Build binary
```

Comment:
keep chapters in chronology - Prereq, build, usage, docker, release. 
No scrolling to bottom for build information